proposals/lip-0054.md

```diff
@@ -452,13 +455,14 @@ def execute(trs: Transaction) -> None:
         eventsDefaultTopic = ccmID
 
         ccm = decode(crossChainMessageSchema, ccmBytes)
-        # If the sending chain is the mainchain, recover the CCM.
+        # If the sending chain is the own chain, recover the CCM.
         # This function never raises an error.
-        if ccm.sendingChainID == getMainchainID():
+        if ccm.sendingChainID == OWN_CHAIN_ID:
             applyRecovery(trs, ccm)
-        # If the sending chain is not the mainchain, forward the CCM.
+        # Sending chain is not the own chain. 
+        # This can happen only in mainchain. We then forward the CCM.
         # This function never raises an error.
-        elif ccm.sendingChainID != getMainchainID():
+        else ccm.sendingChainID != getMainchainID():
```

Comment:
I think this should be elif, unless you remove the following comparison

Reply:
Right, I think with else is better, forgot to remove the comparison. Done.